target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -367,6 +371,10 @@ ramips_setup_macs()
 	wnce2001)
 		wan_mac=$(mtd_get_mac_binary factory 46)
 		;;
+	barionet-1000-v1-1-a9)
```

Comment:
Drop this here. The mac address is set via mtd-mac-address in the dts and according to your commit message you do not have a wan interface.